Less2

ASDemin opened the pull request.

ASDemin committed:
Я так и не понял как считать внутри: 3 и 5 вопросы
+ с заменой ссылок, как то не очень красиво, думаю, что как-то можно заменить...но у меня не получилось

ASDemin committed:
добавил pattern для поиска ссылок

ASDemin committed:
добавил файл текст

ASDemin committed:
спасибо за подсказку с модулем коллекция.
посчитал и слова и символы. ХЗ насколько красиво.

ASDemin committed:
отфильтровал все спаны, но количество учеников так и не нашёл...

ASDemin committed:
Delete text.txt

ASDemin committed:
Delete main.py

ASDemin committed:
надо было просто выйти из личного кабинета)

ASDemin committed:
попытался с регулярными выражениями

DmitryChitalov commented on scripte_re.py:
```diff
@@ -0,0 +1,15 @@
+
+print(re.findall('<span class="total-users">Нас уже ([\w ]+) человек<\/span>',s ))
+
+
```
лучше, чтобы выводилось число, а не массив

DmitryChitalov commented on scrypte_bs.py:
```diff
@@ -0,0 +1,17 @@
+li=soup.find_all('span')
+for i in li:
+    print(i.string)
+    break
```
лучше извлекать только число, а не всю строку
Нас уже 3 737 610 человек

ASDemin committed:
Вроде разобрался с первыми 6 пунктами, но дальше нужна помощь:
если по сути задание у нас есть слова
первое (Привет, существительное)
второе(всем, существительное)

и  мы склеиваем по первое.текст, второе.текст или  надо было
Привет(первое, сущ)
всем(второе, сущ.)
